README.md

```diff
@@ -502,6 +503,37 @@ Supported error codes:
 * `pb_utils.TritonError.UNAVAILABLE`
 * `pb_utils.TritonError.UNSUPPORTED`
 * `pb_utils.TritonError.ALREADY_EXISTS`
+* `pb_utils.TritonError.CANCELLED` (since 23.10)
+
+#### Request Cancellation Handling
+
+One or more requests may be cancelled by the client during execution. Starting
+from 23.10, `request.is_cancelled()` returns up-to-date `True` or `False` on
```

Comment:
`request.is_cancelled()` returns whether the request is cancelled.

Reply:
Updated: [Documentation wording updates](https://github.com/triton-inference-server/python_backend/pull/304/commits/3a43ef3a9d0550935ae9857f3c3d54e73987ab35)